internal/api/v1/applications.go

```diff
@@ -75,7 +77,8 @@ func (hc ApplicationsController) Create(w http.ResponseWriter, r *http.Request)
 	return nil
 }
 
-// Index lists all the known applications, with and without workload.
```

Comment:
Some comments have operations like GET POST mentioned but some are not.

Reply:
Yes. Added the information only when I noticed same url, different http methods. Can be consolidated to always have the method.